src/overlays/actors/ovl_Obj_Roomtimer/z_obj_roomtimer.h

```diff
@@ -3,14 +3,16 @@
 
 #include "global.h"
 
+#define ROOMTIMER_GET_SWITCHFLAG(thisx) (((thisx)->params >> 9) & 0x7F)
+
 struct ObjRoomtimer;
 
 typedef void (*ObjRoomtimerActionFunc)(struct ObjRoomtimer*, GlobalContext*);
 
 typedef struct ObjRoomtimer {
     /* 0x0000 */ Actor actor;
     /* 0x0144 */ ObjRoomtimerActionFunc actionFunc;
-    /* 0x0148 */ char unk_144[0x4];
+    /* 0x0148 */ s32 flag;
```

Comment:
I think I'd rather this were called `switchFlag`, since it's easy to mix up `this->flag` and `thisx->flags`.